content/en/docs/Installation/_index.md

```diff
@@ -456,16 +456,4 @@ kubectl delete -f https://raw.githubusercontent.com/kyverno/kyverno/main/definit
 
 ```sh
 helm uninstall kyverno --namespace kyverno kyverno/kyverno
-```
```

Comment:
These set of backticks need to remain.

Reply:
I haven't removed the backtics. this git diff is kind of misleading. 

check the backticks [here](https://github.com/kyverno/website/pull/253/commits/87d93a1c574ba558b6117119c5c5a222a3b8d831#diff-2304e00d07492b5b6f7e83ed1a4f4050f0ebbbee5d5a2fc3cd4dd30bfe1cfc78R459)